lib/constants/constants.dart

```diff
@@ -6,6 +6,12 @@ const gradientBaseGreen = 0xff2a9d8f;
 
 //PATHS
 
+const orangeLogoPath = "assets/bluelogo.png";
+const RegisterPageGradientPath = "assets/registerpagegradient.png";
+const registerPageImagePath = "assets/registerpageimage.png";
+
 const blueLogoPath = "assets/bluelogo.png";
 const blueGradientPath = "assets/bluegradient.png";
-const loginPageImagePath = "assets/loginpageimage.png";
\ No newline at end of file
+const loginPageImagePath = "assets/loginpageimage.png";
+
+const pattern = r'^(?=.*?[A-Z])(?=.*?[a-z])(?=.*?[0-9])(?=.*?[!@#\$&*~]).{8,}$';
```

Comment:
Rename to be more suggestive. You might have a pattern for something else in the future.